code/nxt/src/logic.c

```diff
@@ -12,8 +12,10 @@ bool laser_state = false;
 void main_loop() {
     T_TARGET_LOCATION target_location;
     if (get_target_location(&target_location)) {
-        target_location.x -= 64;
-        target_location.y -= 48;
+
+        /*target_location.x -= 64;
```

Comment:
Skal bare fjernes, er ikke brugbart når funktionen virker

Reply:
HVIS det virker :P 
men ja. fjerner det. ville bare ikke fjerne det helt for testing purposes. men ja